lumicks/pylake/fitting/tests/test_fd_models.py

```diff
@@ -129,15 +160,15 @@ def test_model_reprs():
 @pytest.mark.parametrize(
     "model, test_params",
     [
-        (model_impl.WLC, ["Lp", "Lc", "St", "kT"]),
-        (model_impl.invWLC, ["Lp", "Lc", "St", "kT"]),
-        (model_impl.FJC, ["Lp", "Lc", "St", "kT"]),
-        (model_impl.invFJC, ["Lp", "Lc", "St", "kT"]),
-        (model_impl.marko_siggia_simplified, ["Lp", "Lc", "kT"]),
-        (model_impl.marko_siggia_ewlc_solve_force, ["Lp", "Lc", "St", "kT"]),
-        (model_impl.marko_siggia_ewlc_solve_distance, ["Lp", "Lc", "St", "kT"]),
-        (model_impl.tWLC, ["Lp", "Lc", "St", "kT"]),
-        (model_impl.invtWLC, ["Lp", "Lc", "St", "kT"]),
+        (model_impl.ewlc_odijk_iforce, ["Lp", "Lc", "St", "kT"]),
+        (model_impl.ewlc_odijk_distance, ["Lp", "Lc", "St", "kT"]),
+        (model_impl.efjc_distance, ["Lp", "Lc", "St", "kT"]),
+        (model_impl.efjc_solve_iforce, ["Lp", "Lc", "St", "kT"]),
+        (model_impl.wlc_marko_siggia_force, ["Lp", "Lc", "kT"]),
+        (model_impl.ewlc_marko_siggia_force, ["Lp", "Lc", "St", "kT"]),
+        (model_impl.ewlc_marko_siggia_distance, ["Lp", "Lc", "St", "kT"]),
```

Comment:
Why is it that the `ewlc_marko_siggia` has no 'inverted' version?

Reply:
Original reason is that independent and dependent variable are on both sides of the equation. After a discussion with @JoepVanlier we came to the conclusion to remove the `i` everywhere in favor of an even cleaner naming convention, as the `i` indicates an unimportant implementation detail to the user.

Reply:
Maybe such details could be discussed in a theory section.